INSTALL.md

```diff
@@ -38,10 +38,37 @@ Dangerzone is available for:
   ```
 </details>
 
-Add our repository following [these instructions](https://packagecloud.io/firstlookmedia/code/install#manual-deb), or by running this script:
+Add our repository following these instructions:
 
+Install `curl`, if it's not already available:
+
+```sh
+sudo apt-get install -y curl
+```
+
+Download the GPG key for the repo:
+
+```sh
+sudo mkdir -p /etc/apt/keyrings/
+curl -s https://packages.freedom.press/apt-tools-prod/fpf-apt-tools-archive-keyring.gpg \
+    | sudo tee /etc/apt/keyrings/fpf.gpg >/dev/null
 ```
-curl -s https://packagecloud.io/install/repositories/firstlookmedia/code/script.deb.sh | sudo bash
+
+Verify it against its SHA-256 checksum
```

Comment:
From a user perspective, I don't think providing a checksum here is that reassuring; versus having them run something like `gpg --keyring /etc/apt/keyrings/fpf.gpg --list-keys` and then verify that only a single key is printed, and provide the fingerprint + email here.

Reply:
I've pushed some changes that address Kunal's comment, so I'm resolving this discussion. Feel free to re-open it.